velox/functions/sparksql/registration/RegisterDatetime.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/lib/RegistrationHelpers.h"
+#include "velox/functions/prestosql/DateTimeFunctions.h"
+#include "velox/functions/sparksql/DateTimeFunctions.h"
+
+namespace facebook::velox::functions::sparksql {
+
+void registerDatetimeFunctions(const std::string& prefix = "") {
```

Comment:
ditto. Do we need the default value for prefix, as it should be provided by the top-level registration?

Reply:
ok, add default value in Register.cpp